### PR TITLE
Show an application tab w/ session cookies

### PR DESCRIPTION
**Pull request by jasonLaster**
https://www.loom.com/share/529b476910c141188a23deaae2111a8b

Screenshot 2023-05-20 at 12 24 45 PM

### CODE REVIEW

File: src/ui/components/SecondaryToolbox/Application.tsx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+}
+cookies
+*/
+async function getCookies(replayClient: ReplayClientInterface): Promise<string> {
+  const response = await ThreadFront.evaluate({
+    replayClient: replayClient,
+    text: `document.cookie;`,
+  });
+
+  console.log(response);
```
Comment (bvaughn) [inline src/ui/components/SecondaryToolbox/Application.tsx:47]: Lots of `console.log` statements in here that should be removed

File: src/ui/components/SecondaryToolbox/Application.tsx [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+var cookiesArray = allCookies.split(';');
+var cookies = {};
+for (var i = 0; i < cookiesArray.length; i++) {
+  var cookie = cookiesArray[i].trim();
+  var separatorIndex = cookie.indexOf('=');
+  var cookieName = cookie.substring(0, separatorIndex);
+  var cookieValue = cookie.substring(separatorIndex + 1);
+  cookies[cookieName] = cookieValue;
+}
+cookies
```
Comment (bvaughn) [inline src/ui/components/SecondaryToolbox/Application.tsx:39]: Delete commented out code

**Comment by bvaughn** [comment]
I rebased this so the new Delta will run on it. I also fixed the Lint issue and _one_ of the TypeScript issues. Realized this didn't look ready for review at that point, so I left the other TS issue :)

**Comment by jasonLaster** [comment]
Thanks for tidying this up. This was definitely a POC although if conceptually it looks good, i can get it to the point where it's an experimental feature.

**Comment by bvaughn** [comment]
I didn't really think critically about it 😄  Happy to give it a proper review once it's ready thouhg!

**Comment by bvaughn** [comment]
This PR seems stale so I'm going to close it. Let's re-open when you're ready to pick it back up though!

**Pull request closed without merging**